Doc/library/io.rst

```diff
@@ -908,6 +908,24 @@ Text I/O
 
       Whether line buffering is enabled.
 
+   .. attribute:: write_through
+
+      Whether writes are passed immediately to the underlying binary
+      buffer.
+
+      .. versionadded:: 3.7
+
+   .. method:: reconfigure(*, line_buffering=None, write_through=None)
+
+      Reconfigure this text stream using new values for *line_buffering*
+      and *write_through*.  Any ``None`` value will keep the current
+      value.
```

Comment:
Rather than using `value` 3 times, this may be clearer with a mix of `setting` and `argument`:

> Reconfigure this text stream using new settings for *line_buffering* and *write_through*.  Passing ``None`` as an argument will retain the current setting for that parameter.

That's just a suggestion though, I don't think it's a blocker for merging the patch.